spec/controllers/availability_controller_spec.rb

```diff
@@ -27,6 +27,36 @@
   end
 
   describe '#index with id param' do
+    it "returns availability for a single item" do
+      bib_id = "991227850000541" # Equivalent to 263059 in Voyager
+      stub_alma_ids(ids: bib_id, status: 200, fixture: "unsuppressed_991227850000541")
+
+      get :index, params: { id: bib_id, format: :json }
+      availability = JSON.parse(response.body)
+      # Useful
+      # https://api-na.hosted.exlibrisgroup.com/almaws/v1/bibs/991227850000541?expand=p_avail
+      # https://api-na.hosted.exlibrisgroup.com/almaws/v1/bibs/991227850000541/holdings/all/items
+      # https://bibdata.princeton.edu/availability?id=263059
+      expect(availability).to eq(
+        "2282456310006421" => {
+          "more_items" => false,
+          "location" => "sa",
+          # "copy_number" => 1, TODO => Implement
+          "item_id" => "2382456270006421"
+          # "on_reserve": "N", TODO: Implement
+          # "patron_group_charged": null, TODO: Implement
+          # "status": "On-Site", TODO: Implement
+          # "label": "Marquand Library", TODO: Implement
+          # "status_label": "On-site access" TODO: Implement
+        },
+        # This isn't in Voyager's bibdata, I don't know why.
```

Comment:
@mzelesky Do you know why 991227850000541 has a holding in Alma but not from production bibdata? Is there something about this holding that makes it so it shouldn't show up, or is the data just different?

Reply:
I think we added an item while we were testing.

Reply:
Yes, as @christinach said, the MAIN-main item was added in Alma during testing.